system-int-test/test/cmr/system_int_test/search/granule_search_format_test.clj

```diff
@@ -1,6 +1,7 @@
 (ns cmr.system-int-test.search.granule-search-format-test
   "Integration tests for searching granules in csv format"
   (:require [clojure.test :refer :all]
```

Comment:
Mega nit: the standard we've adopted is to move this first require to a new line and alphabetize the requires.

```
(:require
 [clojure.string :as string]
 [clojure.test :refer :all]
```